src/views/tutorial-view/components/mdx-img/mdx-img.module.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  background-color: var(--token-color-surface-primary);
+  box-shadow: var(--token-surface-base-box-shadow);
+  border-radius: 6px;
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
+  text-align: center;
+  justify-content: center;
+  margin: 24px auto;
+
+  &.noMargin {
+    margin: 0;
+  }
+
+  &.noBorder {
+    box-shadow: none;
+  }
+}
+
+.img {
+  /* TODO: remove content styles which force !important here */
```

Comment:
Added exhaustive properties here, these overlap with `:global` CSS.

The intent here is that we remove [the `:global` css for `img`](https://github.com/hashicorp/dev-portal/blob/78d0143803bd5826bf933f507546111a5bafce11/src/components/dev-dot-content/dev-dot-content.module.css#L85). I'd be happy to handle that in a follow-up PR if everyone's on board with the approach 🙇‍♂️

Reply:
I think removing global image styles for dev dot sounds good! The global styles will probably hurt more than they help imo.

Reply:
I'm in agreement that the styles can and should be moved from `DevDotContent`, we just need to make sure that the styles get applied to all content in the same way it currently is for consistency.